go.mod

```diff
@@ -3,35 +3,21 @@ module github.com/mittwald/harbor-operator
 go 1.15
 
 require (
-	github.com/elazarl/goproxy v0.0.0-20200315184450-1f3cb6622dad // indirect
-	github.com/fatih/color v1.9.0 // indirect
 	github.com/go-logr/logr v0.1.0
-	github.com/go-openapi/jsonreference v0.19.4 // indirect
-	github.com/go-openapi/runtime v0.19.20 // indirect
-	github.com/golang/mock v1.4.3
-	github.com/imdario/mergo v0.3.8
+	github.com/imdario/mergo v0.3.9
 	github.com/jinzhu/copier v0.0.0-20190924061706-b57f9002281a
-	github.com/mattn/go-colorable v0.1.6 // indirect
-	github.com/mittwald/go-helm-client v0.3.0
-	github.com/mittwald/goharbor-client v1.0.4
-	github.com/operator-framework/operator-sdk v0.18.2
+	github.com/mittwald/go-helm-client v0.4.1
+	github.com/mittwald/goharbor-client/v3 v3.0.3
```

Comment:
We should use v3.0.4 to not mess up our dependencies, see https://github.com/mittwald/goharbor-client/pull/68

Reply:
fixed